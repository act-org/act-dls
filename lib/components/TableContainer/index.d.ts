/**
 * @prettier
 */
import * as React from 'react';
import { TableContainerProps } from '@material-ui/core';
declare const TableContainer: React.FC<TableContainerProps>;
export default TableContainer;
//# sourceMappingURL=index.d.ts.map