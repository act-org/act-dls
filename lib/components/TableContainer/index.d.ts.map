{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["../../../src/components/TableContainer/index.tsx"],"names":[],"mappings":"AAAA;;GAEG;AAEH,OAAO,KAAK,KAAK,MAAM,OAAO,CAAC;AAC/B,OAAO,EAEL,mBAAmB,EACpB,MAAM,mBAAmB,CAAC;AAI3B,QAAA,MAAM,cAAc,EAAE,KAAK,CAAC,EAAE,CAAC,mBAAmB,CAIjD,CAAC;AAEF,eAAe,cAAc,CAAC"}