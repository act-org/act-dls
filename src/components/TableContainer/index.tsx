/**
 * @prettier
 */

import * as React from 'react';
import {
  TableContainer as TableContainerImport,
  TableContainerProps,
} from '@material-ui/core';

import TableContainerComponent from '~/components/TableContainerComponent';

const TableContainer: React.FC<TableContainerProps> = (
  props: TableContainerProps,
): React.ReactElement<any> => (
  <TableContainerImport component={TableContainerComponent} {...props} />
);

export default TableContainer;
