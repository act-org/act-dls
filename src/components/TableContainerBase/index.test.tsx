/**
 * @prettier
 */

import * as React from 'react';
import renderer from 'react-test-renderer';

import TableContainer from '.';

describe('TableContainer', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<TableContainer />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
