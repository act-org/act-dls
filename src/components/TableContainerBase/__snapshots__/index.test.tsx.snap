// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableContainer matches the snapshot 1`] = `
<div
  className="MuiTableContainer-root"
/>
`;
